src/parameterized_tendencies/radiation/RRTMGPInterface.jl

```diff
@@ -430,49 +449,9 @@ function RRTMGPModel(
                GrayRadiation"
     end
 
-    if (
-        :center_pressure in keys(dict) &&
-        :center_temperature in keys(dict) &&
-        :face_pressure in keys(dict) &&
-        :face_temperature in keys(dict)
-    )
-        if !(interpolation isa NoInterpolation)
-            @warn "interpolation is ignored if both center and face pressures/\
-                   temperatures are specified"
-        end
-        implied_values = :none
-    elseif (
-        :center_pressure in keys(dict) &&
-        :center_temperature in keys(dict) &&
-        !(:face_pressure in keys(dict)) &&
-        !(:face_temperature in keys(dict))
-    )
-        if interpolation isa NoInterpolation
-            error("interpolation cannot be NoInterpolation if only center \
-                   pressures/temperatures are specified")
-        end
-        implied_values = :face
-    elseif (
-        !(:center_pressure in keys(dict)) &&
-        !(:center_temperature in keys(dict)) &&
-        :face_pressure in keys(dict) &&
-        :face_temperature in keys(dict)
-    )
-        if interpolation isa NoInterpolation
-            error("interpolation cannot be NoInterpolation if only face \
-                   pressures/temperatures are specified")
-        end
-        implied_values = :center
-    else
-        error("please specify either center_pressure and center_temperature, \
-               or face_pressure and face_temperature, or all four values")
-    end
-
-    if implied_values != :face
-        if !(bottom_extrapolation isa SameAsInterpolation)
-            @warn "bottom_extrapolation is ignored if face_pressure and \
-                   face_temperature are specified"
-        end
+    if interpolation isa NoInterpolation
```

Comment:
I think we can remove `NoInterpolation` too.

Reply:
I'll do that in the next one.